src/pages/Components/RatingStars/index.md

```diff
@@ -0,0 +1,47 @@
+---
+title: 'Rating Stars'
+order: 1
+---
+
+> Rating stars is a pattern you'll find in almost every website. It allows to do two different things. First, it allows the user to give a feedback on his experience with a product or a service. Second, it allows a brand to provide some informations on other user's opinion on a product or a service to make his choice easier.
+
+<preview path="src/pages/Components/RatingStars/previews/stars-input-basic" nude=true></preview>
+<preview path="src/pages/Components/RatingStars/previews/stars-result-basic" nude=true></preview>
+
+## Variations
```

Comment:
Rating stars is a component composed of 5 stars, the following text seems to indicate that rating stars is a single star. 

I think the following is not very important. 

we have separated ratting stars into `mc-stars-input` and `mc-stars-results`, I think we should have separated definitions and documentation (in the same page) about both of this elements with rules like : 

### mc-stars-input

`mc-stars-input` allow user to rate a product or a service. 
user can only rate using integer from 0 to 5, by clicking on the corresponding star

do: `mc-stars-input` should always have a label that indicate the user he is able to rate the product
dont: `mc-stars-input` should never display half stars


### mc-stars-result

`mc-stars-result` display notes from 0 to 5 and allow alf values (2.5) that are the result of user rated products or services
`mc-stars-result` is composed of 5 stars that can either represant empty, full or alf full states.

Reply:
And there is something bothering me about the design... nothing set appart the input from the result in term of design

Reply:
@gael-boyenval I tried a version that takes your suggestion into account